algorithms-and-data-structures/basic-javascript/index050.js

```diff
@@ -0,0 +1,6 @@
+let xp = 0;
+let health = 100;
+let gold = 50;
+let currentWeapon = 0;
+
+// Now you will declare a variable without intializing it. Using the let keyword, declare a variable called fighting but don't set it equal to anything. Just end the line with a simicolon right after the varible name. 
```

Comment:
Add ` `` ` backticks to the `fighting` word

Spelling:
- `intializing`->`initializing` 
- `simicolon`->`semicolon`
- `varible`->`variable`
```suggestion
/*
Now you will declare a variable without intializing it.
Using the let keyword, declare a variable called `fighting` but don't set it equal to anything.
Just end the line with a semicolon right after the variable name. 
```